src/services/SendATFReport.ts

```diff
@@ -25,25 +25,31 @@ class SendATFReport {
    * @param generationServiceResponse - The response from the ATF generation service
    * @param visit - Data about the current visit
    */
-  public async sendATFReport(generationServiceResponse: any, visit: any): Promise<any> {
+  public async sendATFReport(generationServiceResponse: any, visit: any) {
     // Add testResults and waitActivities in a common list and sort it by startTime
     const activitiesList = this.computeActivitiesList(generationServiceResponse.testResults, generationServiceResponse.waitActivities);
 
     const response = await this.testStationsService.getTestStationEmail(visit.testStationPNumber);
+    console.debug(`response from get test station email: ${JSON.stringify(response)}`);
     const sendNotificationData = this.notificationData.generateActivityDetails(visit, activitiesList);
+    console.debug(`send notification data : ${JSON.stringify(sendNotificationData)}`);
     if (!this.notifyService) {
       if (!this.apiKey) {
         this.apiKey = (await Configuration.getInstance().getGovNotifyConfig()).api_key;
       }
       this.notifyService = new NotificationService(new NotifyClient(this.apiKey));
     }
+
+    let combinedEmails = []
```

Comment:
The visit.testerEmail should always be populated, so you could assign this variable to that right here, rather than line 50? And then add the testStationEmails if they exist.

Reply:
nice, didn't know that was an assumption, will sort that now

Reply:
Yes I mean in the current code there is an assumption that it is populated anyway, I guess there would be no way of sending them certificates that had been generated or the atf-report otherwise.